src/api/emit.js

```diff
@@ -20,19 +31,48 @@ function createCustomEvent (name, opts) {
   return e;
 }
 
+function createReadableStopPropagation (oldStopPropagation) {
+  return function () {
+    this.isPropagationStopped = true;
+    oldStopPropagation.call(this);
+  };
+}
+
+function simulateBubbling (elem, cEvent) {
+  var status;
+  cEvent.stopPropagation = createReadableStopPropagation(cEvent.stopPropagation);
+  while (elem && !cEvent.isPropagationStopped) {
+    cEvent.currentTarget = elem;
+    if (elem.dispatchEvent(cEvent) === false) {
+      status = false;
```

Comment:
I suggest renaming `status` to `didPreventDefault` (or similar) to make this a little easier to understand.